service/src/main/java/uk/gov/hmcts/reform/fpl/utils/DocumentsHelper.java

```diff
@@ -0,0 +1,27 @@
+package uk.gov.hmcts.reform.fpl.utils;
+
+import uk.gov.hmcts.reform.fpl.enums.DocumentStatus;
+import uk.gov.hmcts.reform.fpl.model.common.Document;
+
+public class DocumentsHelper {
+
+    private DocumentsHelper() {
+        // No OP
+    }
+
+    public static boolean hasDocumentPresent(Document document) {
+        return document != null;
+    }
+
+    public static boolean hasDocumentStatusSet(Document document) {
+        return hasDocumentPresent(document) && document.getDocumentStatus() != null;
+    }
+
+    public static boolean hasDocumentStatusOf(Document document, DocumentStatus documentStatus) {
+        return hasDocumentStatusSet(document) && document.getDocumentStatus().equals(documentStatus.getLabel());
+    }
+
+    public static boolean hasDocumentUploaded(Document document) {
+        return document.getTypeOfDocument() != null;
```

Comment:
Shouldn't we check hasDocumentPresent here, if someone invokes that directly might throw nullpointer :( 

Otherwise I really like it.